code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -248,10 +248,15 @@
 	..()
 	spawn(2400)
 		if(!owner && !preserved)
-			inert = 1
-			desc = "The remains of a hivelord that have become useless, having been left alone too long after being harvested."
+			go_inert()
 		else
-			preserved = 1
+			preserved = TRUE
+			feedback_add_details("hivelord_core", "[src.type]|implanted")
```

Comment:
It indicates that the legion heart has been implanted in a person, and been stabilised by merit of being implanted.